app/src/main/kotlin/model/internalGraphs/_WeightedDirectedGraph.kt

```diff
@@ -20,4 +21,49 @@ abstract class _WeightedDirectedGraph<D, E : WeightedDirectedEdge<D>> : _Directe
      * In case weight is not passed, set it to default value = 1
      */
     override fun addEdge(vertex1: Vertex<D>, vertex2: Vertex<D>) = addEdge(vertex1, vertex2, 1)
+
+    fun findShortestPath(srcVertex: Vertex<D>, destVertex: Vertex<D>): List<Pair<Vertex<D>, E>> {
```

Comment:
Bellman-Ford algorithm also finds shortest path, method should be renamed.

Reply:
Changed to 'findShortestPathDijkstra')